fuzzers/utils.py

```diff
@@ -61,7 +61,8 @@ def append_flags(env_var, additional_flags, env=None):
     '-pthread', '-Wl,--no-as-needed', '-Wl,-ldl', '-Wl,-lm',
     '-Wno-unused-command-line-argument'
 ]
-NO_SANITIZER_COMPAT_CXXFLAGS = ['-stdlib=libc++'] + NO_SANITIZER_COMPAT_CFLAGS
+NO_SANITIZER_COMPAT_CXXFLAGS = ['-stdlib=libc++', '-pthread'] + \
```

Comment:
I don't want to make you spend more time debugging linking issues, but I'm curious why this change was necessary.
1. Why don't you use these variables?
2. Is the change to CXXFLAGS necessary given this change?
3. Why does the flag need to be added here? Is what we were doing before to append not sufficient?
4. Can you change this to
```
NO_SANITIZER_COMPAT_CXXFLAGS = (['-stdlib=libc++', '-pthread'] +
   NO_SANITIZER_COMPAT_CFLAGS)
```
We prefer using paren to backslash to break up lines in python.